test/CMFE/vec_init/vec_init_diag.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: not %cmc -emit-llvm -- %s 2>&1 | FileCheck %s
+
+#include <cm/cm.h>
+
+_GENX_ void test_func() {
+  vector<unsigned short, 4> v1 = {};                    // OK
+  vector<unsigned short, 4> v2 = {1, 2};                // OK
+  vector<unsigned short, 4> v3 = {1, 2, 3, 5};          // OK
+  vector<unsigned short, 4> v4 = {1, 2, 3, 5, 7, 11};   // error: excess elements
+  vector<unsigned short, 4> v5 = {{1}};                 // warning: braces around scalar
+  vector<unsigned short, 4> v6 = {{1}, 2, 3, 5, 7, 11}; // warning: braces around scalar; error: excess elements
+// CHECK: vec_init_diag.cpp:9:47: error: excess elements in CM vector initializer
+// CHECK: vec_init_diag.cpp:10:35: warning: braces around scalar initializer
+// CHECK: vec_init_diag.cpp:11:35: warning: braces around scalar initializer
+// CHECK: vec_init_diag.cpp:11:49: error: excess elements in CM vector initializer
```

Comment:
Is it possible to also cover "err_init_conversion_failed" diagnostic? I see that you modified it so I think there should be a case in this test that triggers this error.

Reply:
Fixed.